contrib/natss/pkg/reconciler/dispatcher/natsschannel.go

```diff
@@ -131,7 +153,9 @@ func (r *Reconciler) Reconcile(ctx context.Context, key string) error {
 	}
 
 	// Try to subscribe.
-	if err := r.natssDispatcher.UpdateSubscriptions(c, false); err != nil {
+	subStatus, err := r.natssDispatcher.UpdateSubscriptions(c, false)
+	natssChannel.Status.SubscribableStatus = subStatus
```

Comment:
is it correct to set the value even if there was an error? 
Shouldn't you check for the error first? If not, maybe add a comment, as a future maintainer might get confused